src/Integration/RequestFetcher.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Sentry\Integration;
+
+use GuzzleHttp\Psr7\ServerRequest;
+use Psr\Http\Message\ServerRequestInterface;
+
+/**
+ * Default implementation for RequestFetcherInterface. Creates a request object from the PHP superglobals.
```

Comment:
Would you be so kind to split this description so that it desn't last longer than ~80 chars per line?